tests/Jwt/FirebaseParserTest.php

```diff
@@ -51,32 +50,28 @@ protected function setUp()
         ];
     }
 
+    /**
+     * @expectedException \Equip\Auth\Exception\InvalidException
+     * @expectedExceptionCode \Equip\Auth\Exception\InvalidException::CODE_TOKEN_EXPIRED
```

Comment:
@elazar This version of PHPUnit requires PHP 5.6. The Equip still supports PHP 5.5 (Travis CI).

[Supported Versions of PHP](https://secure.php.net/supported-versions.php)